### ISSUE TITLE
Option to disable (or enable) Earth rotation

### ISSUE THREAD

**Issue by stevage** [TerriaJS/terriajs#1519]
When displaying time series animations, the spinning earth (and hence, rotating star field) is distracting and unhelpful. Once you enable Cesium lighting, it can become almost unusable, as the day/night cycle flies around the earth.

IMHO, the Earth should _not_ rotate by default, and an option in `config.json` should be required to enable it:

```
{
   "parameters": {
    "earthRotation": true   
   }
}
```

**Comment by meh9** [comment]
I agree. AREMI has some timeseries data (the ISF Network Opportinities) that spans many years with only a single datapoint per year. Because of that we play the timeseries very very fast, and it is indeed unhelpful to have a flickering star field behind the globe when playing.

**Comment by kring** [comment]
Random idea: it might be cool - rather than disabling lighting and the sun and stars entirely - to instead synchronize time steps to the same time each day.  That way you would see the stars move, but the whole experience would be less seizure-inducing.

**Comment by stevage** [comment]
Hmm. I guess my feeling is that Terria is not an astronomical simulator, and people don't expect it to be. It's jarring and confusing when people discover what's going on. They're using the site to look at stuff on the ground, and anything happening in the sky is just distracting and potentially confusing. </curmudgeon>

**Comment by kring** [comment]
Yeah I agree it's not really necessary.  But turning off the animation of the moon and starts, without turning them off completely, will probably be challenging.  I don't think they're inherently a problem except that they flicker like crazy when animating quickly.  My suggestion would be pretty straightforward to implement (it's basically just an optional rounding in the timestep) and would probably be accepted into Cesium itself.
